tests/backend/test_renew.py

```
@@ -0,0 +1,150 @@
+			DATABASE_URL = f"sqlite:///{db_path}"
+			TESTING = True
+
+		self.app = create_app(RenewTestConfig)
+		self.client = self.app.test_client()
+		SQLModel.metadata.create_all(get_engine())
+
+		# Create test user
+		with Session(get_engine()) as session:
+			self.test_user = User(
+				email="test@example.com",
+				password_hash="hashed_password",
```

Ruff (0.15.4)

[error] Possible hardcoded password assigned to argument: "password_hash"

(S106)